src/metabase/models/dashboard_card.clj

```diff
@@ -375,3 +376,23 @@
       (update :created_at             #(if (string? %) (u.date/parse %) %))
       (update :parameter_mappings     serdes/import-parameter-mappings)
       (update :visualization_settings serdes/import-visualization-settings)))
+
+(defn- dashboard-card-series-xform
+  [ingested]
+  (-> ingested
+      (update :card_id          serdes/*import-fk* :model/Card)
+      (update :dashboardcard_id serdes/*import-fk* :model/DashboardCard)))
+
+(defmethod serdes/load-one! "DashboardCard"
+  [ingested maybe-local]
+  (let [dashcard ((get-method serdes/load-one! :default) (dissoc ingested :series) maybe-local)]
+    ;; drop all existing series for this card and recreate them
+    ;; TODO: this is unnecessary, but it is simple to implement
```

Comment:
Ah, I remarked on this elsewhere. Since this is a P1 escalation I'm content with this for now, since there's a `TODO` for future gardening to notice and improve this.